src/Redistribution.sol

```diff
@@ -538,14 +538,18 @@ contract Redistribution is AccessControl, Pausable {
     function currentRoundAnchor() public view returns (bytes32 returnVal) {
         uint256 cr = currentRound();
 
+        // This will be called in reveal phase and set as currentRevealRoundAnchor or in
+        // commit phase when checking eligibility for next round by isParticipatingInUpcomingRound
         if (currentPhaseCommit() || (cr > currentRevealRound && !currentPhaseClaim())) {
             return currentSeed();
         }
 
+        // This will never be called  ¯\_(ツ)_/¯
```

Comment:
this is for sanity check

Reply:
you can remove it, but then leave a comment with the clarification that the check tries to achieve.